icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PersonNameFormatterTest.java

```diff
@@ -355,7 +364,7 @@ public void TestNameSpacing() {
             // the native formatter
             new NameAndTestCases("locale=ja_JP,given=駿,surname=宮崎", new String[][] {
                 { "ja_JP", "LONG",   "REFERRING",  "FORMAL",   "DEFAULT", "", "宮崎駿" },
-                { "zh_CN", "LONG",   "REFERRING",  "FORMAL",   "DEFAULT", "", "宮崎 駿" },
+                { "zh_CN", "LONG",   "REFERRING",  "FORMAL",   "DEFAULT", "", "宮崎駿" },
```

Comment:
I want to call your attention specifically to this test case.  The earlier change to always use the formatter's own patterns and to use `nativeSpaceReplacement` or `foreignSpaceReplacement` messed up this example, with my getting back "宮崎・駿".  And, of course, we were also getting the dot in "Albert Einstein".  I fixed "Albert Einstein" to not get the dot, but the logic I added (using the native formatter for the name) doesn't apply to this case, which still got the dot.  I changed the `hasSameScript()` method so that Japanese and Chinese would return true, and that got me "宮崎駿", with no dot and also no space.  I don't think there's a good way to get the space in this ("宮崎 駿"), and I'm not sure we want it.  But if we do, there's more work to do here.  Please let me know what you think.

Reply:
We discussed this in the Person Names meeting, and we all agreed this is indeed the result we want.